src/panels/lovelace/cards/hui-picture-glance-card.ts

```diff
@@ -256,11 +256,17 @@ class HuiPictureGlanceCard extends LitElement implements LovelaceCard {
             `
           : html`
               <div class="state">
-                ${computeStateDisplay(
-                  this.hass!.localize,
-                  stateObj,
-                  this.hass!.language
-                )}
+                ${entityConf.attribute
+                  ? html`
+                      ${entityConf.prefix}${stateObj.attributes[
```

Comment:
What happens if you specify an attribute that doesn't exist

Reply:
you lose

Reply:
You could do `stateObj.attributes[entityConf.attribute] || "-"`